l2tdevtools/dependency_writers/check_dependencies.py

```diff
@@ -14,12 +14,21 @@ class CheckDependenciesWriter(interface.DependencyFileWriter):
 
   _TEMPLATE_DIRECTORY = os.path.join('data', 'templates')
 
+  _PROJECTS_WITH_PYTHON3_AS_DEFAULT = ('plaso', )
```

Comment:
l2tscaffolder also has a python3 as default, please add it here.

Reply:
I don't think l2tdevtools is currently used for l2tscaffolder, but I'll add it